README.md

```diff
@@ -105,50 +110,119 @@ ___
+            between an successful subscription and an unsuccessful subscription with the Thing just not emitting an event, so the
+            subscription test has essentially three different outcomes: Successful, Timeout and Failed. Successful describes the case
+            where node-wot confirms a successful subscription, Timeout describes the case where node-wot can not differentiate (see
+            above) and Failed describes the case where node-wot throws an error during subscription (The timeout length can be configured in
+            the testConfig).
+    -   The listening phase
+        -   The testbench listens a specified amount of time for any incoming data for this subscription. Any received data is verified
+            against output specified in the TD.
+        -   The listening phase ends after a configurable amount of time or when a configurable amount of data packages was received
+            (both options can be configured in the testConfig).
+    -   The cancel subscription test (depends on the outcome fot the subscription test)
+        -   If subscription test was successful the testbench sends the specified request for unsubscribeEvent.
+        -   If subscription test was a timeout, the testbench can not know if the subscription was successful so it does not test
+            anything.
+        -   If subscription test was a fail the testbench can obviously not cancel the subscription so it does not test anything.
+
+-   The test request is returned with the current state of the testReport.
+-   The testReport property is updated with the current state of the testReport
+
+### Synchronous listening Phase (only if events or observable properties are present; optional)
+
+-   Can be explicitly deactivated in the testConfig.
+-   All Events and observable properties are tested again but this time synchronously. This synchronous testing needs significantly less
+    time but results in a pretty hard to read log.
+-   The timeout length, listening length and the received data package threshold can all be configured independent of the listening phase of
+    the sequential tests in the testConfig.
 
-- This link provides all possible postman interaction examples [https://documenter.getpostman.com/view/4378601/RWEmHGBq](https://documenter.getpostman.com/view/4378601/RWEmHGBq).
+### Ending Phase
 
-- How to use testbench screencast video can be found here [https://youtu.be/BDMbXZ2O7KI](https://youtu.be/BDMbXZ2O7KI).
+-   The finished testReport is written to the storage
+-   If the Synchronous listening phase was present the testReport property is updated to the current state.
+-   The testbench is reset to be ready for the next test run.
 
-- You can use your browser and the GET requests to inspect all properties during the procedure.
+### TLDR
 
+<img src="readme-images/test.svg" align="center" height="400" width="400">

```

Comment:
@egekorkan I added a readme-images folder on root is this ok, or should this be changed? When using svg for images I noticed, I can only give size information if using html instead of markdown directly...

Reply:
That is fine :+1: